src/farbfeld.rs

```diff
@@ -101,6 +101,13 @@ impl<R: Read> Read for FarbfeldReader<R> {
     }
 }
 
+impl<R: Read + Seek> Seek for FarbfeldReader<R> {
+    fn seek(&mut self, pos: SeekFrom) -> io::Result<u64> {
+        self.cached_byte = None;
+        self.inner.seek(pos)
```

Comment:
I don't think this is right. You shouldn't be able to seek before the start of the pixel data, and if the seek into the middle of a channel cached_byte shouldn't be None.

Reply:
You're right, see top commit for fixed implementation